scalableminds/webknossos
# Uses local images for logos and background

### Steps to test:
- Check logo on spotlight view (maybe set `features.allowOrganizationCreation = true`)

------
~- [ ] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)~
~- [ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~
~- [ ] Updated [documentation](../blob/master/docs) if applicable~
~- [ ] Needs datastore update after deployment~
